data/RockingSNP.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "Souvik Pramanick",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Full Stack Web Developer **Digital Nomad**. Explporer & Karma -te Prastut",
+
+"links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/SouvikP74860962",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/RockingSNP/",
+      "icon": "FaGithub"
+    },
+  {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/souvik-p-117647218/",
+      "icon": "FalinkedIn"
+    },
+   {
+      "name": "Checkout my works on replit",
+      "url": "https://replit.com/@RockingSNP",
+      "icon": "FaReplit"
+    },
+{
+      "name": "Join our Enthusiastic Open-source Napptive Community on Slack",
+      "url": "https://join.slack.com/t/napptivecommunity/shared_invite/zt-1ov6ocst2-BsZDhfOZYAet86D6TxlIkg",
+      "icon": "FaSlack"
+    },
+{
+      "name": "Or Follow Us on Twitter",
+      "url": "https://www.twitter.com/NapptiveCompany",
+      "icon": "FaTwitter"
+    },
+ {
+      "name": "Subscribe me on Youtube",
+      "url": "https://www.youtube.com/@souvikpramanick1549/",
+      "icon": "FaYoutube"
+    },
+{
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/rockingsp_official/",
+      "icon": "FaInstagram"
+    },
```

Also remove this extra comma `,` from here. 

```suggestion
    }
```